autofill/autofill-impl/src/main/java/com/duckduckgo/autofill/ui/credential/saving/AutofillSavingCredentialsDialogFragment.kt

```diff
@@ -68,6 +69,7 @@ class AutofillSavingCredentialsDialogFragment : BottomSheetDialogFragment(), Cre
     }
 
     private fun configureViews(binding: ContentAutofillSaveNewCredentialsBinding, credentials: LoginCredentials) {
+        (dialog as BottomSheetDialog).behavior.state = BottomSheetBehavior.STATE_EXPANDED
```

Comment:
Couldn't find a way to do this from the styles

Reply:
Just to clarify, this is the expanded state? and not fully occupying the screen?
![rn_image_picker_lib_temp_22692eaf-91e3-4bbb-a496-ae73f2e89924](https://user-images.githubusercontent.com/2943941/185653103-f16b9776-293b-4e33-be04-7cb965688102.jpg)

Reply:
Fully expanded **vertically**. Before it would only peek up a little and the CTA button would be down offscreen below the fold

So yeah, that's what we want now.